src/core/OrbitControls.tsx

```diff
@@ -7,19 +7,24 @@ export type OrbitControlsProps = ReactThreeFiber.Overwrite<
   {
     target?: ReactThreeFiber.Vector3
     camera?: THREE.Camera
+    domElement?: HTMLElement
     regress?: boolean
     enableDamping?: boolean
   }
 >
 
 export const OrbitControls = React.forwardRef<OrbitControlsImpl, OrbitControlsProps>(
-  ({ camera, regress, enableDamping = true, ...restProps }, ref) => {
+  ({ camera, regress, domElement, enableDamping = true, ...restProps }, ref) => {
     const invalidate = useThree(({ invalidate }) => invalidate)
     const defaultCamera = useThree(({ camera }) => camera)
     const gl = useThree(({ gl }) => gl)
     const performance = useThree(({ performance }) => performance)
     const explCamera = camera || defaultCamera
-    const controls = React.useMemo(() => new OrbitControlsImpl(explCamera), [explCamera])
+    const explDomElement = domElement || gl.domElement
+    const controls = React.useMemo(
+      () => new OrbitControlsImpl(explCamera, explDomElement),
```

Comment:
on L39, we may no longer need to `controls.connect(gl.domElement)` if we are connecting to the dom element in the constructor. L39 can likely be deleted with this change

Reply:
@haywirez if we can implement this, we can merge this today. Sorry for the delay!

Reply:
Thanks for catching this, just force pushed the update with the changes 👍